### PR TITLE
Polishing the docs

### PR DESCRIPTION
**Pull request by Mxrcon**
Hey, I took some time to continue the work started by @Aryan-sharma11 and decided to add
a deploy to heroku button and some other specifications for the deployment instructions.

You can check the overall modifications on`README`.

Before moving to a merge, I'd like to hear your opinions and overall suggestions for this package.

Kindly, Davi

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -41,7 +41,7 @@ RUN micromamba install --yes --name base \
 # Clean up all other packages which are not necessary
 RUN micromamba clean --all --yes
 
-COPY --chown=$MAMBA_USER:$MAMBA_USER ./assets/profile.ps1 /home/mamba/.config/powershell/profile.ps1
+COPY --chown=$MAMBA_USER:$MAMBA_USER resources/asssets/profile.ps1 /home/mamba/.config/powershell/profile.ps1
```
Comment (abhi18av) [inline Dockerfile:44]:
```suggestion
COPY --chown=$MAMBA_USER:$MAMBA_USER resources/assets/profile.ps1 /home/mamba/.config/powershell/profile.ps1
```

File: README.md [hunk 2]
```diff
@@ -17,31 +24,79 @@ This project aims to be a Proof of Concept of APIs using tools useful for bioinf
...
+```powershell
+docker build -t sucuri-api 
+docker run -p 0.0.0.0:8085:8085 sucuri-api:latest
+```
 
-### Using a container (Locally and Heroku)
 
 ## Usage
+Sucessfuly depolying the api, will expose it to the 8085 port, which you can use to make 
+api calls.
```
Comment (abhi18av) [inline README.md:70]:
```suggestion
Sucessfuly depolying the api, will expose it to the 8085 port, which you can use to make api calls.

```

File: README.md [hunk 3]
```diff
@@ -17,31 +24,79 @@ This project aims to be a Proof of Concept of APIs using tools useful for bioinf
...
 
+### Parameters
+
+Api request url example:
```
Comment (abhi18av) [inline README.md:74]:
```suggestion
API request url example:

```

**Review by abhi18av — APPROVED** [review]
This PR looks good to merge (with some minor changes).

**Comment by abhi18av** [comment]
Great work @Mxrcon ! 

@Aryan-sharma11 , we have a great project for future work 🚀

**Comment by Aryan-sharma11** [comment]
> Great work @Mxrcon !
> 
> @Aryan-sharma11 , we have a great project for future work 🚀

Yes...  this is looking really good🥳

**Pull request merged**